code/api.tf

```diff
@@ -126,6 +132,11 @@ data "azurerm_key_vault_secret" "backend_client_secret" {
   key_vault_id = azurerm_key_vault.key_vault.id
 }
 
+data "azurerm_key_vault_secret" "backend_geolocation_token" {
+  name         = "backend-GEOLOCATION-TOKEN"
```

Comment:
@AleDore ..... one more .... where do I get the value to put in this secret?
What is it the GEOLOCATION-TOKEN?

Reply:
@uolter I will provide it later :)